crates/polars-time/src/chunkedarray/rolling_window/dispatch.rs

```diff
@@ -82,8 +82,9 @@ where
     if ca.is_empty() {
         return Ok(Series::new_empty(ca.name(), ca.dtype()));
     }
-    let ca = ca.rechunk();
-    let by = by.rechunk();
+    if by.null_count() > 0 {
+        polars_bail!(InvalidOperation: "'Expr.rolling_*_by(...)' not yet supported for series with null values, consider using 'DataFrame.rolling' or 'Expr.rolling'")
+    }
```

Comment:
this needs checking for `ca` too? looks like it's being removed from L107 but not replaced

Reply:
also, L114 can probably now just become
```rust
        let by_values = by.cont_slice().expect("`by` has already been rechunked and checked for null values");
```
?

likewise L137

Reply:
It isn't I believe, but we should check that higher up indeed. :+1:

Reply:
Ah, it is. NVM.